presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkConfig.java

```diff
@@ -165,4 +166,16 @@ public PrestoSparkConfig setSparkBroadcastJoinMaxMemoryOverride(DataSize sparkBr
         this.sparkBroadcastJoinMaxMemoryOverride = sparkBroadcastJoinMaxMemoryOverride;
         return this;
     }
+
+    public boolean isSmileSerializationEnabled()
+    {
+        return smileSerializationEnabled;
+    }
+
+    @Config("spark.smile-serialization-enabled")
+    public PrestoSparkConfig setSmileSerializationEnabled(boolean smileSerializationEnabled)
```

Comment:
Do we need this config property?

Reply:
After discussion with @tdcmeehan it looks like Smile should be 100% compatible with Json in theory, in practice we've had issues in the past. Although we've done several iterations to improve compatibility and the likelihood of us seeing issues for Split serialization is rather minimal I would still prefer to have a fallback if something goes awry.